util/tryGetEnv.node.ts

```diff
@@ -0,0 +1,9 @@
+declare global {
+  interface Process {
+    env: Record<string, string>
+  }
+  export const process: Process | undefined
+}
+export function tryGetEnv(name: string): string | undefined {
+  return typeof process !== "undefined" ? process.env?.[name] : undefined
```

Comment:
@tjjfvi why this instead of `process?.env?.[name]`?

Reply:
`process?.env?.[name]` will error if `process` is not defined, but this will not.